# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../elftoaout/prefix-usage.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.orig	2004-06-27 01:18:28.370283816 +0200
+++ ./Makefile	2004-06-27 01:25:31.611941304 +0200
@@ -1,5 +1,6 @@
 CC = gcc
 CFLAGS = -O2 -Wall
+PREFIX = /usr
 
 all: elftoaout
 
@@ -7,8 +8,8 @@
 	$(CC) $(CFLAGS) -o elftoaout elftoaout.c
 
 install: elftoaout
-	install -m 0755 -s elftoaout /usr/bin/elftoaout
-	install -m 0644 elftoaout.1 /usr/man/man1/elftoaout.1
+	install -m 0755 -s elftoaout $(PREFIX)/bin/elftoaout
+	install -m 0644 elftoaout.1 $(PREFIX)/man/man1/elftoaout.1
 
 clean:
 	rm -f *~ *.[soa] core
